src/rqt_launch/node_controller.py

```diff
@@ -49,33 +49,33 @@ def start(self, restart=True):
                 # If the node is already running and restart isn't desired,
                 # do nothing further.
                 return
-            #TODO: Need to consider...is stopping node here
+            # TODO: Need to consider...is stopping node here
             # (i.e. in 'start' method) good?
             self.stop()
 
         # If the launch_prefix has changed, then the _process must be recreated
         if (self._proxy.config.launch_prefix !=
-            self._gui._lineEdit_launch_args.text()):
+                self._gui._lineEdit_launch_args.text()):
 
             self._proxy.config.launch_prefix = \
-                                     self._gui._lineEdit_launch_args.text()
+                self._gui._lineEdit_launch_args.text()
             self._proxy.recreate_process()
 
         self._gui.set_node_started(False)
         self._gui.label_status.set_starting()
         self._proxy.start_process()
         self._gui.label_status.set_running()
         self._gui.label_spawncount.setText("({})".format(
-                                              self._proxy.get_spawn_count()))
+            self._proxy.get_spawn_count()))
```

Comment:
This line doesn't need to be broken

Reply:
The break is necessary to keep it under 79 characters as specified in PEP 8.

Reply:
Python guide for ROS1 doesn't have a line length, and defaults to ROS C++ guide which is 120. But this is a pull request for preparation for porting to ROS 2, which has a 100 character line length.

Reply:
If that is the case, then should we actually be applying autopep8 at all? Is there a configuration file for ROS 2?

Reply:
Can you share a link to the style guide for ROS 1? I only know of this: http://wiki.ros.org/PyStyleGuide

That guide specifies PEP 8 but lists a few exceptions.

Is the goal to actually use this repo for the ROS 2 version of `rqt_launch` or to do a fork to avoid breaking backward compatibility. Based on the significant overhaul to the launch system in ROS 2, I would think we would want a fork which means we probably wouldn't want to apply the ROS 2 style guide to this ROS 1 package. Thoughts, @brawner?

Reply:
I guess the PyStyleGuide doesn't really say what I was mentioning. I had just heard line lengths were adopted from C++ standards. If this package uses 79, then we should just stick with that.

For now, we'll be creating crystal-devel branches in RQT. But the packages that should be forked vs branched are not entirely resolved. So it's good to hear some comments about it.

Reply:
Cool, let me know when this gets resolved, I'm happy to support the decision.

Reply:
Thanks for this improvement!

I can't speak for ROS2 branching. That said, and while I'm afraid of being pedantic, [PEP8 defines 79 character length](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), which I believe I tried to stick to as much as possible when I made this (experimental) package. Back then there was a discussion for the line length at Willow for Python code, which didn't seem to have ended up making change in the [wiki.ros.org/PyStyleGuide](http://wiki.ros.org/PyStyleGuide). So I'm +1 for 79 limit wherever that makes sense.